src/components/begrunnelse/begrunnelse.tsx

```diff
@@ -219,15 +219,15 @@ const Begrunnelse = (props: any) => {
                 </Normaltekst>
             </MarginContainer>
 
-            <MarginContainer>
-                <CenteredContainer>
+            <DoubleMarginTopContainer>
+                <DoubleMarginTopContainer>
```

Comment:
Er det nødvendig å doble disse? Det er litt pirk, men det virker litt rart at man ikke kan oppnå det man ønsker i en enkel div.

Reply:
> Er det nødvendig å doble disse? Det er litt pirk, men det virker litt rart at man ikke kan oppnå det man ønsker i en enkel div.

Ja, kan legge det i en enkeltdiv, er nok bedre for UU feks